train.py

```diff
@@ -12,176 +12,30 @@
-    drop_rate = hyperparameters.drop_rate
-    n_epoch = hyperparameters.total_epoch
-    n_save = hyperparameters.save_every
-    ################################
-
-    train_loader, val_loader = get_loaders()
-
-    num_classes = len(REF_CLASSES)
-
-    model = timm.create_model(config.model_name, pretrained=True, in_chans=config.num_channels, num_classes=num_classes,
-                              drop_rate=drop_rate).to(device)
-
-    print(f"============================ MODEL {config.model_name} ============================")
-
-    best_auc = 0
-    best_val_loss = 1e10
-
-    for param in model.parameters():
-        param.requires_grad = False
-
-    for param in model.classifier.parameters():
-        param.requires_grad = True
-
-    # Unfreeze the last N blocks
-    for block in model.blocks[-hyperparameters.num_blocks_to_unfreeze:]:
-        for param in block.parameters():
-            param.requires_grad = True
-
-    optimizer = torch.optim.Adam(filter(lambda p: p.requires_grad, model.parameters()),
-                                 lr=hyperparameters.learning_rate, weight_decay=weight_decay)
-
-    criterion = nn.CrossEntropyLoss()
-    scheduler = torch.optim.lr_scheduler.ReduceLROnPlateau(
-        optimizer,
-        mode=scheduler_params.mode,
-        patience=scheduler_params.patience,
-        factor=scheduler_params.factor,
-    )
+checkpoint_name = None
 
-    logs = {
-        "epoch": [],
-        "epoch_time": [],
-        "train_loss": [],
-        "val_loss": [],
-        "val_auc": [],
-        "val_accuracy": [],
-    }
-    for epoch in range(n_epoch):
-        start = time.time()
-        print(f"\n================\nEpoch {epoch + 1}")
-        model.train()
-        train_loss = 0.0
-        for imgs, labels in train_loader:
-            imgs, labels = imgs.to(device), labels.to(device)
-            outputs = model(imgs)
-            loss = criterion(outputs, labels)
-            loss.backward()
-            torch.nn.utils.clip_grad_norm_(model.parameters(), max_norm=max_norm, norm_type=2)
-            optimizer.step()
-            optimizer.zero_grad()
-            train_loss += loss.item()
 
-        train_loss = train_loss / len(train_loader)
-        print(f"\nTrain Loss: {train_loss:.4f}")
+def parse_run_arguments() -> None:
+    global checkpoint_name
 
-        model.eval()
-        val_preds, val_labels = [], []
-        val_loss = 0.0
-        with torch.no_grad():
-            for imgs, labels in val_loader:
-                imgs, labels = imgs.to(device), labels.to(device)
-                outputs = model(imgs)
-                loss = criterion(outputs, labels)
-                val_loss += loss.item()
-                probs = torch.softmax(outputs, dim=1)
-                val_preds.append(probs.cpu().numpy())
-                val_labels.extend(labels.cpu().numpy())
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-c', '--continue_training', action="store", default=None)
```

Comment:
If continue flag requires a checkpoint path, maybe rename it to:
`load_checkpoint` (and then, I'd NOT allow the short `-c` arg)